src/Firehose.Web/Firehose.Web.csproj

```diff
@@ -16,10 +16,6 @@
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <UseIISExpress>true</UseIISExpress>
     <IISExpressSSLPort>44371</IISExpressSSLPort>
-    <IISExpressAnonymousAuthentication />
```

Comment:
Please don't change any of these settings

Reply:
I'll add this again - a mistake when reverting the changes that VS made on compiling/testing the app.

Reply:
It is not only this line, which you added back, it is the whole blog. Check your commits.